### PR TITLE
Update grafana dashboards

### PR DESCRIPTION
**Pull request by jan--f**
This PR fixes a number of dashboard issues.

Some panels didn't display data due to metric renames in the both the mgr module and the node exporter.
The style is more uniform: stacking fill are used where it makes sense, line width is uniform, panels that show read and write metrics display read as negative, legends and units are fixed.
`label_replace` call are overhauled.

Fixes: http://tracker.ceph.com/issues/39652

### CODE REVIEW

**Comment by LenzGr** [comment]
Thanks for fixing these! Adding the Dashboard label, being a consumer of these dashboards.

File: monitoring/grafana/dashboards/cephfs-overview.json [hunk 1]
```diff
@@ -77,7 +78,12 @@
       "pointradius": 5,
       "points": false,
       "renderer": "flot",
-      "seriesOverrides": [],
+      "seriesOverrides": [
+        {
+          "alias": "/.*Reads/",
+          "transform": "negative-Y"
```
Comment (epuertat) [inline monitoring/grafana/dashboards/cephfs-overview.json]: Interesting approach. I tried this once and had the impression that I was losing detail due to the increase in the vertical span of the graph. I haven't tried with recent Grafana releases: is the horizontal axis kept centered or it scales from bottom to top?

Comment (jan--f) [reply monitoring/grafana/dashboards/cephfs-overview.json]: One certainly looses granularity but on the other hand gains more clarity which direction data flows. I'd argue that these panels are rarely used to get accurate measurements of say an ingress rate (though that is certainly still an option) but rather to see if there's a heavy read or write workload currently stressing the cluster.

The y-axis still scales depending on the data, i.e. with no Reads in the above panel the 0 point should be at the bottom of the panel.

**Comment by LenzGr** [comment]
Small suggestion: would it make sense to squash all commits into one and add a `Fixes: http://tracker.ceph.com/issues/39652` comment to the git commit message? This will make it easier to backport this into the nautilus branch.

**Comment by smithfarm** [comment]
Follow-up question to Lenz's - since the backport need is already reflected in the tracker, is there any need for the "needs-backport" label? (Note: the backporting team does not use this label in its workflows, so it's very presence here in github is a potential source of confusion...)

File: monitoring/grafana/dashboards/pool-detail.json [hunk 2]
```diff
@@ -36,22 +37,257 @@
...
+      "sparkline": {
+        "fillColor": "rgba(31, 118, 189, 0.18)",
+        "full": false,
+        "lineColor": "rgb(31, 120, 193)",
+        "show": false
+      },
+      "tableColumn": "",
+      "targets": [
+        {
+          "expr": "(-ceph_pool_stored / deriv(ceph_pool_max_avail[6h])) * on(pool_id) group_left(instance,name) ceph_pool_metadata{name=~\"$pool_name\"}",
```
Comment (a2batic) [inline monitoring/grafana/dashboards/pool-detail.json]: Looks good to me. Addition in Ceph Pool Details looks very nice. Just one question:- I am observing the negative value in `Time till full` panel. Is it due to a difference in setup configuration ?
Screenshot from 2019-05-10 15-33-29

Comment (jan--f) [reply monitoring/grafana/dashboards/pool-detail.json]: Ah yes...good catch. Had a formula error, should be fixed now. I also added a `> 0` and mapped null to `inf`.

**Comment by callithea** [comment]
jenkins retest this please

File: monitoring/grafana/dashboards/pool-detail.json (hunk 2 above)
Comment (a2batic) [reply monitoring/grafana/dashboards/pool-detail.json]: Great, thanks!

**Comment by LenzGr** [comment]
Note that this PR changed the UUIDs of some dashboards, which breaks embedding them into the Ceph Dashboard. See #28169 for a fix.

**Pull request merged**